src/main/java/org/fundacion/jala/converter/core/facade/ConverterFacade.java

```diff
@@ -10,10 +10,12 @@
  */
 package org.fundacion.jala.converter.core.facade;
 
+import org.fundacion.jala.converter.core.exceptions.ConverterException;
 import org.fundacion.jala.converter.core.parameter.AudioParameter;
 import org.fundacion.jala.converter.core.parameter.VideoParameter;
 import org.fundacion.jala.converter.core.AudioConverter;
 import org.fundacion.jala.converter.core.VideoConverter;
+
```

Comment:
jessi, this empty line should be removed

Reply:
Thanks, I have just fixed it